### PR TITLE
[D2IQ-62644] Percona-pxc-mysql - Migrate artifacts to downloads.mesosphere.com

### PR DESCRIPTION
**Pull request by farhan5900**
Resolves [D2IQ-62644](https://jira.mesosphere.com/browse/D2IQ-62644)

## What changes were proposed in this pull request?
Migrated the S3 bucket artifacts to downloads.mesosphere.com to avoid the issues after S3 bucket deletion. Hence changing the existing resource-uris, images paths.

## How were these changes tested?
- Successfully deployed the Percona-pxc-mysql service on DCOS 2.0 cluster.
- Soaked on soak200s, soak201s clusters.

### CODE REVIEW

**Comment by ankitcid** [comment]
Hi @farhan5900 , could you please check for the failure Check _(Mesosphere repo validation (version-3.x) )_.

**Comment by farhan5900** [comment]
Hi @ankitcid It was due to download failure of some SBT dependencies. Resolved by running it again.

**Review by rishabh96b — APPROVED** [review]
LGTM :tada:  Let's merge this after one more approval.

**Review by ankitcid — APPROVED** [review]
Thanks LGTM!

**Pull request merged**
